src/_posts/2020-06-01-signs-your-software-is-rotting.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+asset-type: post
+name: signs-your-software-is-rotting
+title: Signs Your Software is Rotting
+date: 2020-06-09 00:00:00 +00:00
+author: Matt Belcher
+description: In this article, we explore why software rot is an issue, outline some tell-tale signs that your software is in decline, and some ways you can address it.
+abstract: In this article, we explore why software rot is an issue, outline some tell-tale signs that your software is in decline, and some ways you can address it.  
+image:
+   src: /assets/custom/img/blog/2020-06-01-signs-your-software-is-rotting/software-rotting.jpg
+   attribution:
+      text: Photo by Karl Pawlowicz on Unsplash
+      href: https://unsplash.com/photos/QUHuwyNgSA0
```

Comment:
I dislike having the photo attribution be front and center, is there another place we can have the photo attribution.